### PR TITLE
Upgrade to webpack 4 and add dev-server

### PR DESCRIPTION
**Pull request by marcodejongh**

### CODE REVIEW

**Comment by conorhastings** [comment]
it doesn't look like the ExampleLinks file is included

**Comment by marcodejongh** [comment]
@conorhastings Oops, checked in now

File: demo/prism-async-light.js [hunk 1]
```diff
@@ -0,0 +1,146 @@
+import React from 'react';
+import { render } from 'react-dom';
+import { PrismAsyncLight as SyntaxHighlighter} from '../src';
```
Comment (conorhastings) [inline demo/prism-async-light.js]: is this import correct, i get an error trying to access this demo

```
Uncaught TypeError: Cannot read property 'registerLanguage' of undefined
    at Object../demo/prism-async-light.js (prism-async-light.js:9)
    at __webpack_require__ (bootstrap:724)
    at fn (bootstrap:101)
    at Object.3 (prismAsyncLight-build.js:52820)
    at __webpack_require__ (bootstrap:724)
    at ./demo/examples-links.js.Object.defineProperty.value (bootstrap:791)
    at bootstrap:791
```

**Comment by marcodejongh** [comment]
Merging this as it's only a devtool change and helps me cleanup my other PR's

**Pull request merged**